extension/src/popup/metrics/authServices.ts

```diff
@@ -0,0 +1,10 @@
+import { METRIC_NAMES } from "popup/constants/metricsNames";
```

Comment:
this file need to be imported somewhere to actually run this code and register the handler right?

Reply:
Ah good point. I meant this to be a complete example so I'll make sure this is used

Reply:
cool - just making sure i understood what the implementation would look like